### PR TITLE
fixed test Remove Reliability tags

### LINKED ISSUE #5431: [Test] Remove Reliability tags from test/config.xml
This module was deleted so we don't need this information in that file.

### PR DESCRIPTION
**Pull request by VictorVermeulen**
fixes #5431 

#### Testing instructions (if applicable)

1.

#### Links to related tickets (GitHub, Redmine, ...)

*

### CODE REVIEW

**Comment by VictorVermeulen** [comment]
Fixes #5431

File: docs/deprecated_wiki/Setup/Initial Setup/Security/Security:-Enabling-SSL.md [hunk 1]
```diff
@@ -1,20 +0,0 @@
-**[[HOME|Home]]** > **[[SETUP|Setup]]** > **[[SECURITY: ENABLING SSL|Security: Enabling SSL]]**
```
Comment (driusan) [inline docs/deprecated_wiki/Setup/Initial Setup/Security/Security:-Enabling-SSL.md]: it looks like you accidentally deleted this file, which isn't related to the ticket you're trying to fix

**Comment by johnsaigle** [comment]
Thanks for issuing a PR @VictorVermeulen! Your changes to the configuration file look good and ready to merge.

As @driusan mentioned, please undo your changes to the Markdown files. After that we'll be good to go.

**Comment by VictorVermeulen** [comment]
I am not able to do this yet but I am trying with the help of someone.

When I try this: https://riptutorial.com/git/example/2444/restore-a-deleted-file-after-a-commit

I get this screen: https://i.imgur.com/1wJkG32.png It stays like that and I can never execute the second command. I did restart multiple times and also did try git add ''file path'' but that did not work either. 

Also tried this: https://i.imgur.com/1Wq8rQN.png

Maybe you know something is wrong here? Also if I get it right, the deleted part in the config file was correct? I thought I deleted too much code? Thanks in advance.

**Comment by johnsaigle** [comment]
Could you try running the following command:

`git checkout [COMMIT] -- my-file.txt docs/deprecated_wiki/Setup/Initial Setup/Security/Security docs/deprecated_wiki/Setup/Initial Setup/Security/Security:-Enabling-SSL.md`

But replace `[COMMIT]` with the commit hash from `git log`. In this case it should be the long string of text beside the word `commit` authored by Xavier in your second screenshot.

This command will restore the contents of the files to what they were before your commit.

https://stackoverflow.com/questions/7147270/hard-reset-of-a-single-file

Let me know if you have any more questions and I'll help further.

**Comment by VictorVermeulen** [comment]
This does not work either. The file is missing apparently. 

https://i.imgur.com/vlItMKB.png

**Comment by johnsaigle** [comment]
Hmm okay. in that case do you think you could redo this PR again and be careful to add only the changes relating to the `config.xml` file? It'll likely be simpler to do that than to debug the problem further.

**Pull request closed without merging**